pinot-common/src/test/java/org/apache/pinot/common/metrics/AbstractMetricsTest.java

```diff
@@ -48,4 +49,63 @@ public void testAddOrUpdateGauge() {
     controllerMetrics.removeGauge(metricName);
     Assert.assertTrue(controllerMetrics.getMetricsRegistry().allMetrics().isEmpty());
   }
+
+  @Test
+  public void testConcurrentGaugeUpdates() throws InterruptedException {
```

Comment:
Since we are not doing concurrent updates, can you rename the method as well. Something like: `testMultipleUpdatesToGauge`

Reply:
Thank you for the review. Fixed!